docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -73,7 +73,7 @@ For ease of use, the date picker will automatically change the layout between po
 
 ## Sub-components
 
-Some lower-level sub-components (`CalendarPicker`, `MonthPicker`, and `YearPicker`) are also exported. These are rendered without a wrapper or outer logic (masked input, date values parsing and validation, etc.).
+Some lower-level sub-components (`CalendarPicker`, `MonthPicker`, and `YearPicker`) are also exported.
```

Comment:
I don't see the link with this PR

Reply:
> date values parsing

There is no parsing any more, that's the part related to this PR

> validation

Not related to this PR but this is false, they do have validation

> masked input

Indeed, but I don't think it's worth specifying the the `MonthPicker` does not have a masked input, it's obvious by looking at the example below